File: sampleCode/CSharp/ConsoleApp/Endpoints/Regions/CombineRegionRequest.cs
namespace ConsoleApp.Regions;

public class CombineRegionRequest
{
    /// <summary>
    /// A unique-per-Project description for this Combined Region
    /// </summary>
    public string Description { get; set; }

    /// <summary>
    /// An array of the HashIds for all the Regions to be combined
    /// </summary>
    public string[] HashIds { get; set; } = [];

    /// <summary>
    /// An array of the Urids for all the Regions to be combined
    /// </summary>
    public long[] Urids { get; set; } = [];
}

File: sampleCode/Python/endpoints/Regions/CombineRegionRequest.py
class CombineRegionRequest:
    def __init__(self, description, hashids, urids=None):
        self.description = description
        self.hash_ids = hashids
        self.urids = urids if urids is not None else []

    def to_dict(self):
        return {
            "description": self.description,
            "hashIds": self.hash_ids,
            "urids": self.urids
        }
